lend/src/utils/asset.ts

```diff
@@ -0,0 +1,133 @@
+import React from "react";
```

Comment:
this can also be imported from `shared/lib/utils/asset` @timongll

Reply:
the `VaultOptions` and `Chains` used here is different